From 58af83399f6e86f7851cfe9d33b7006c25e9534e Mon Sep 17 00:00:00 2001
From: S-trace <S-trace@list.ru>
Date: Tue, 27 Dec 2016 10:25:09 +0300
Subject: [PATCH] [CMSDK] Fix build failure

org.cyanogenmod.platform-sdk depends on org.cyanogenmod.platform-res
---
 Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Android.mk b/Android.mk
index dc2b714..8d27929 100644
--- a/Android.mk
+++ b/Android.mk
@@ -132,6 +132,9 @@ LOCAL_INTERMEDIATE_SOURCES := \
 LOCAL_JAVA_LIBRARIES := \
     $(cmsdk_LOCAL_JAVA_LIBRARIES)
 
+LOCAL_ADDITIONAL_DEPENDENCIES += \
+    org.cyanogenmod.platform-res
+
 # Make sure that R.java and Manifest.java are built before we build
 # the source for this library.
 cm_framework_res_R_stamp := \
-- 
2.9.3

